addons/mail/static/src/discuss/call/common/call_action_button.js

```diff
@@ -0,0 +1,38 @@
+import { Component, xml } from "@odoo/owl";
+
+/**
+ * @typedef {Object} Props
+ * @extends {Component<Props, Env>}
+ */
+export class CallActionButton extends Component {
+    static template = xml`
+        <button class="btn smaller d-flex m-1 border-0 rounded-circle shadow-none opacity-75-hover align-items-center p-0"
+            t-att-class="{ 'bg-600 opacity-75': !props.action.available, 'opacity-100': props.action.available }"
+            t-att-aria-label="props.action.name"
+            t-att-title="title"
+            t-on-click="props.action.select">
+            <i class="fa fa-fw m-2 fa-fw" t-att-class="{
+                'fa-lg': !props.isSmall,
+                [props.action.inactiveIcon]: !props.isActive,
+                [props.action.icon]: props.isActive or !props.action.inactiveIcon,
+                [props.action.activeClass]: props.isActive,
+            }"/>
+        </button>
+    `;
+    static components = {};
+    static props = [
+        "action",
+        "isSmall",
+        "isActive?",
+    ];
+
+    setup() {
+        super.setup();
+    }
```

Comment:
useless method

Reply:
This might be useful unfortunately for `prepareObserveRenders` test helper? If not, then indeed can simply be removed as this comes useless.